### PR TITLE
Master to develop 2214

### PR DESCRIPTION
**Pull request by udaydenduluri33**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: Jenkinsfile_nightly [hunk 1]
```diff
@@ -15,15 +15,16 @@ properties([
...
     enableSlackNotifications(channel)
-
+    enableSecurityScan()
     enableFullFunctionalTest(30)
-
-    env.TEST_URL = 'https://xui-ao-webapp-aat.service.core-compute-aat.internal/'
+    enableMutationTest()
+    enableCrossBrowserTest()
+    env.TEST_URL = 'https://administer-orgs.aat.platform.hmcts.net'
     after('checkout') {sh 'yarn cache clean'}
```
Comment (paddybasi) [inline Jenkinsfile_nightly:28]: just a side note but we should really be using YarnBuilder for these commands

File: debug.log [hunk 2]
```diff
@@ -0,0 +1 @@
+[0526/162334.492:ERROR:prune_crash_reports.cc(35)] PruneCrashReportDatabase: Failed to get pending reports
```
Comment (paddybasi) [inline debug.log:1]: ideally we need to .gitignore this file

File: test/accessibility/reporter/Report.html [hunk 3]
```diff
@@ -0,0 +1,215 @@
...
+    </div>
+
+    
+    </div>
+        <script src="./report_output.js" ></script>
+
+        <script>
+            var data = replacejsoncontent;
+
+            var app = angular.module('myApp', []);
```
Comment (paddybasi) [inline test/accessibility/reporter/Report.html:205]: any particular reason we're using angular v1?

File: test/e2e/config/fullfunctional.conf.js [hunk 4]
```diff
@@ -0,0 +1,118 @@
...
+    //   '../features/**/questions.feature',
+    //   '../features/**/recentEvents.feature',
+    // ],
+    baseUrl: process.env.TEST_URL || 'http://localhost:3000',
+    params: {
+        serverUrls: process.env.TEST_URL || 'http://localhost:3000',
+        targetEnv: argv.env || 'local',
+        // username: process.env.TEST_EMAIL,
+        // password: process.env.TEST_PASSWORD,
+        username: 'vmuniganti@mailnesia.com',
```
Comment (paddybasi) [inline test/e2e/config/fullfunctional.conf.js:64]: again might be test users but I don't think we should have any credentials stored in the repo

File: test/accessibility/reporter/Report.html (hunk 3 above)
Comment (sreekanthpuligadda) [reply test/accessibility/reporter/Report.html:205]: @paddybasi no specific reason.this is custom HTML report we created. We using for Model view control only with angularJS in html file.

**Pull request merged**